tests/devicefactory_test.py

```diff
@@ -746,6 +746,16 @@ def test_get_free_disk_space(self, *args):
     def test_normalize_size(self, *args):  # pylint: disable=unused-argument
         super(LVMVDOFactoryTestCase, self).test_normalize_size()
 
+    @patch("blivet.formats.lvmpv.LVMPhysicalVolume.formattable", return_value=True)
+    @patch("blivet.formats.lvmpv.LVMPhysicalVolume.destroyable", return_value=True)
+    @patch("blivet.static_data.lvm_info.blockdev.lvm.lvs", return_value=[])
+    @patch("blivet.devices.lvm.LVMVolumeGroupDevice.type_external_dependencies", return_value=set())
+    @patch("blivet.devices.lvm.LVMLogicalVolumeBase.type_external_dependencies", return_value=set())
+    @patch("blivet.devices.lvm.LVMVDOPoolMixin.type_external_dependencies", return_value=set())
+    @patch("blivet.devices.lvm.LVMVDOLogicalVolumeMixin.type_external_dependencies", return_value=set())
+    def test_lv_unique_name(self, *args):  # pylint: disable=unused-argument,arguments-differ
+        super(LVMVDOFactoryTestCase, self).test_normalize_size()
```

Comment:
I guess this should be `super(...).test_lv_unique_name()`, right?

Reply:
Oops, fixed.